nucleus/prediction.py

```diff
@@ -365,8 +365,8 @@ class CategoryPrediction(CategoryAnnotation):
     def __init__(
         self,
         label: str,
-        taxonomy_name: str,
         reference_id: str,
+        taxonomy_name: Optional[str] = None,
```

Comment:
Heads up that changing argument order can be a breaking change. A hack to get around this is to manually validate required things are passed in (i.e. reference id). Fine in this case though since it's a new API, just FYI.

Reply:
Doc string needs to be updated here as well to indicate taxonomy_name is optional

Reply:
It actually will autoupdate here from the function arguments. I ended up specifying it for `CategoryAnnotation` because it wouldn't autopopulate in the docs when I initially wrote the docstrings. @drakejwong  do you have any idea why that was the case?